README.md

```diff
@@ -1,2 +1,80 @@
-# iOS_Toolbox
-iOS_Toolbox
+![Shakuro iOS Toolbox](title_image.svg)
+<br><br>
+
+![Version](https://img.shields.io/badge/version-0.5.4-blue.svg)
+![Platform](https://img.shields.io/badge/platform-iOS-lightgrey.svg)
+![License MIT](https://img.shields.io/badge/license-MIT-green.svg)
+
+Toolbox contains various components written in Swift.
+
+- [Components List](#components-list)
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [License](#license)
+
+## Components List
+
+- [Device](/Device/)
+    - **DeviceType** - Helper for detecting model of the device / simulator.
+    - **DeviceOrientationListener** - Alternative for UIDevice.current.orientation.
+- [Extensions](/Extensions/) - Various extensions with small helper functions.
+- [ImageProcessing](/ImageProcessing/)
+    - **ImageProcessor** - Helper for CGImage and UIImage
+- [Keychain](/Keychain/)
+    - **KeychainWrapper** - easy add/remove/get Codable object to/from Keychain.
+- [Keyboard](/Keyboard/)
+    - **KeyboardHandler** - wrapper around keyboard notifications
+- [PlaceholderTextView](/PlaceholderTextView/) - UITextView subclass with a placeholder feature and ability to change own size depending on text contents
```

Comment:
A UITextView subclass with a placeholder feature and the ability to change own size depending on the text contents.